migrations/m160118_081152_add_timetosend_column.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use yii\db\Migration;
+
+class m160118_081152_add_timetosend_column extends Migration
+{
+    public function up()
+    {
+        $this->addColumn('mail_queue', 'time_to_send', $this->dateTime()->notNull());
```

Comment:
We must use `$this->addColumn(Yii::$app->get(MailQueue::NAME)->table, 'time_to_send', $this->dateTime()->notNull());` here so that migration can run successfully. Apply to other places.